Week1/Homework/array.js

```diff
@@ -0,0 +1,7 @@
+let items = [];
+console.log([]);
+console.log(items);
+let animals = ['horse', 'sheep', 'cow'];
+console.log(animals);
+animals.push('Piglet');
+console.log(animals);
```

Comment:
`const` instead of `let` here please since we are not reassigning these variables anywhere 🙂